examples/data_util.py

```diff
@@ -82,9 +85,9 @@ def preprocessing_datasets(datasets, tokenizer, model_name, max_seq_length=1024)
     text_column_name = "text" if "text" in column_names else column_names[0]
 
     if tokenizer.model_max_length < max_seq_length:
-        raise ValueError(
+        logger.warn(
```

Comment:
These changes are covered by #107? I think a rebase can be useful to remove those changes.